azurerm/resource_arm_cognitive_account.go

```diff
@@ -100,6 +100,18 @@ func resourceArmCognitiveAccount() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"key1": {
```

Comment:
whilst I appreciate this is named `key1` in the API - could we name this `primary_access_key` for consistency with the other resources?

Reply:
Updated properties, docs and test